modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/pom.xml

```diff
@@ -15,10 +15,37 @@
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 #if (${liferayVersion.startsWith("7.0")})
 		<liferay.workspace.bundle.url>https://releases-cdn.liferay.com/portal/7.0.6-ga7/liferay-ce-portal-tomcat-7.0-ga7-20180507111753223.zip</liferay.workspace.bundle.url>
+		<targetPlatformVersion>7.0.6</targetPlatformVersion>
```

Comment:
For this property lets match the gradle.properties name... `liferay.workspace.target.platform.version`

Reply:
do we need to add a new required property to let user can set different value of target platform version?